homework/index.js

```diff
@@ -30,15 +32,138 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
-      }
-      createAndAppend('pre', root, { text: JSON.stringify(repositories, null, 2) });
+  function openNewTab(url) {
+    window.open(url, '_blank');
+  }
+
+  function bindContributions(contributorsUrl) {
+    const repoInfoContainer = document.getElementById('repo-info-container');
+    const contributorsDiv = createAndAppend('div', repoInfoContainer, {
+      class: 'contributors-info',
+    });
+    fetchJSON(contributorsUrl)
+      .then(response => {
+        const div = createAndAppend('div', contributorsDiv);
+        createAndAppend('h4', div, { text: 'Contributions' });
+        createAndAppend('hr', div);
+
+        const table = createAndAppend('ul', div, { class: 'grid' });
+        const contributors = response;
+        contributors.forEach((_contributor, index) => {
```

Comment:
Please lookup again how to use the `.forEach()` method. The first argument of the callback function is the current element of the loop iteration. For this particular code snippet, you do not need the `index` parameter:

```js
contributors.forEach(contributor => {
  const imageUrl = contributor.avatar_url;
  const name = contributor.login;
  const contributionsCount = contributor.contributions;
  const profileURL = contributor.html_url;
  ...
```